code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -193,6 +193,12 @@
 	if((src.loc) && isturf(src.loc))
 		if(!stat && !resting && !buckled)
 			for(var/mob/living/simple_animal/mouse/M in view(1,src))
+				if(istype(M, /mob/living/simple_animal/mouse/brown/Tom) && (name == "Jerry")) //Turns out there's no jerry subtype.

```

Comment:
cats named jerry can't catch mice named tom? I feel like this is backwards

Reply:
It's intentionally backwards as per how it is in game right now.
The Cartoon cat is Tom, and the cartoon mouse is jerry.
Our mouse is named Tom, and the cat we have on donut or pubby is Jerry.